Content.Server/_Goobstation/WeaponRandomExplode/WeaponRandomExplodeSystem.cs

```diff
@@ -29,17 +29,26 @@ private void OnShot(EntityUid uid, WeaponRandomExplodeComponent component, ShotA
             if (_random.Prob(component.explosionChance))
             {
                 var intensity = 1;
+                var reduction = 1;
+                if (component.reduction > 1)
+                {
+                    reduction = Convert.ToInt32(component.reduction);
```

Comment:
![image](https://github.com/user-attachments/assets/7bd16019-7e94-4aaf-b65a-89d8eba4da76)

Reply:
HEY JOHN SPACE, DID YOU KNOW YOU CAN MAKE A VARIABLE EITHER EQUAL SOMETHING, OR NOTHING?

Reply:
also what is this supposed to be doing, you can just get it directly from the component and if you need to convert to an int you can just cast it

Reply:
i will eat you

Reply:
lol lmao